templates/vite/package.json

```diff
@@ -9,7 +9,9 @@
     "build": "pnpm run xprebuild && tsc -b && vite build",
     "lint": "eslint .",
     "fuels:dev": "fuels dev",
-    "test": "vitest"
+    "test": "run-s test:*",
+    "test:e2e": "vitest",
+    "test:ui": "./test/ui/test-ui.sh"
```

Comment:
Questions:

1. Are we also running the UI tests in CI for the templates?
2. How long does it take to run them? In CI but also locally.

Reply:
1. Only for the `vite` (default) template
2. Approx 7minutes in both